Samples/iOS-Swift/iOS-Swift/AppDelegate.swift

```diff
@@ -69,20 +68,19 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
             options.enableAppLaunchProfiling = args.contains("--profile-app-launches")
 
+            options.enableAutoSessionTracking = !args.contains("--disable-automatic-session-tracking")
             options.sessionTrackingIntervalMillis = 5_000
             options.attachScreenshot = true
             options.attachViewHierarchy = true
        
 #if targetEnvironment(simulator)
-            options.enableSpotlight = true
-            options.environment = "test-app"
+            options.enableSpotlight = !args.contains("--disable-spotlight")
 #else
-            options.environment = "device-tests"
```

Comment:
I think these environment settings were always effectively overwritten below by setting different values in options.initialScope?